src/mdo/java/WrapperProperties.java

```diff
@@ -350,28 +325,32 @@ public synchronized void loadFromXML(InputStream in) throws IOException, Invalid
 
     @Override
     public void storeToXML(OutputStream os, String comment) throws IOException {
-        Properties props = new Properties();
-        props.putAll(getter.get());
+        OrderedProperties props = new OrderedProperties(getter.get());
```

Comment:
please inline, considering sonar motivation.

- https://github.com/pmd/pmd/issues/5770

Reply:
The variable is neither returned or thrown, it does not apply here.